schema.gql

```diff
@@ -2,6 +2,12 @@
 # THIS FILE WAS AUTOMATICALLY GENERATED (DO NOT MODIFY)
 # ------------------------------------------------------
 
+type Users {
+  firstName: String
+  isMentor: Boolean
+  skills: [String!]
+}
+
```

Comment:
@09brsv não entendi o pq do novo Type, essa task basicamente fazer um findMany na tabela de user que ja existe, porém, com a possibilidade de aplicar um filtro de firstName, isMentor e Skills.

Hoje já temos uma query que faz um get nos users, porém para mentores. Agora nessa iremos abranger todos os tipos de user(sendo mentor ou não) e também poderemos aplicar filtros de busca como parâmetros da query

Reply:
![image](https://github.com/Mentor-Cycle/mentor-cycle-be/assets/57500163/d8489591-ac22-42f8-96b3-a09411684298)
de uma olhada, nessa query de eventos podemos aplicar um filtro de eventos por aprendiz(learnerId) e eventos por mentor(mentorId)
![image](https://github.com/Mentor-Cycle/mentor-cycle-be/assets/57500163/68c2b367-a075-4905-ab2c-5e110ba49a48)
depois na hora de chamar a função do prisma que busca no banco, a gente passa esses paramentros no "where"